examples/Vanilla/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
Awesome addition 👍

Reply:
Shouldn't this also include onload implementation?

Reply:
it have onload (line 16)

Reply:
OK updated